src/services/util.js

```diff
@@ -1,7 +1,21 @@
+import {
+  getLocalStorage,
+  TOKEN_KEY,
+  USER_DATA_KEY,
+  isBrowser,
+} from './localStorage'
 export const getYoutubeThumbnail = url => {
   if (url && url.match(/youtube/)) {
     const id = url.match(/v=(.+)$/)[1]
 
     return `https://i.ytimg.com/vi/${id}/mqdefault.jpg`
   }
 }
+
+const user = getLocalStorage(USER_DATA_KEY)
+
+export const isLoggedIn = () => {
+  return getLocalStorage(TOKEN_KEY) ? true : false
+}
+
+export const getUser = () => (isBrowser && user ? JSON.parse(user) : {})
```

Comment:
this is not quite right .. the variable `user` should be inside the function, so that it is evaluated at the time when `getUser` is called not when the file is imported. This might not be an issue now, because we refresh on login/logout, but we can not rely on that.

Reply:
Changed it but I don't understand..What refreshing has to do with that?

Reply:
so if the variable is outside the function .. that means that it is evaluated once when the script is loaded, which happens once when the page is loaded (or refreshed). If it is inside the function, then it will be evaluated when that function is called, which can be in response to any event, not just page laod (on click, on whatever...)